cli/command/plugin/create.go

```diff
@@ -40,7 +40,7 @@ func validateConfig(path string) error {
 
 // validateContextDir validates the given dir and returns abs path on success.
 func validateContextDir(contextDir string) (string, error) {
-	absContextDir, err := filepath.Abs(contextDir)
+	absContextDir, _ := filepath.Abs(contextDir)
```

Comment:
@AkihiroSuda
```
absContextDir, _ := filepath.Abs(contextDir)
stat, err := os.Lstat(absContextDir)
```
`err` is overwritted without any check. Do you think `err` nave have to alive even if variable is not used ?